server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.0.json

```diff
@@ -22,14 +21,17 @@
     },
     "components": [
         {
+            "catalog_dir": "observability/logs",
```

Comment:
this should be configured on a higher level configuration and all should use that implicit as default location